schema/snapcraft.yaml

```diff
@@ -361,16 +364,20 @@ properties:
           prepare:
             type: string
             default: ''
+          parse-info:
+            type: array
```

Comment:
The array items are always going to be strings, right? We should probably flesh this schema definition out to enforce that.